cubewalkers/tests/test_cana_import.py

```diff
@@ -0,0 +1,20 @@
+# Cubewalkers import
+from cubewalkers import parser as cwp 
+
+# CANA import
+import cana.boolean_network as cana_bn
```

Comment:
please move the import to be inside the test function

Reply:
Done